src/shared/helper.js

```diff
@@ -39,3 +39,17 @@ export const getDate = (day) => {
   const date = new Date();
   return date.addDays(day).toISOString().split("T")[0];
 };
+
+// Returns the initials of given name
+export const getNameInitials = (name) => {
+  // Convert name into array of words
+  const nameList = name.split(" ");
+  // For single word name return first 2 characters
+  if (nameList.length === 1) {
+    return nameList[0].substring(0, 2).toUpperCase();
+  }
+  return nameList
+    .map((n) => n[0])
```

Comment:
Please add a limit of 3 characters otherwise a group name like "a very long group name for testing ui" will break the look of the button.
![image](https://user-images.githubusercontent.com/18077542/126641101-3abebaa1-6e95-4935-a9fc-63b94c50d1d9.png)

Reply:
@GMishx Updated it